tests/phpunit/test-class-wc-stripe-upe-payment-method.php

```diff
@@ -317,4 +352,23 @@ public function test_payment_methods_are_reusable_if_cart_contains_subscription(
 			}
 		}
 	}
+
+	/**
+	 * Test the type of payment token created for the user.
+	 */
+	public function test_create_payment_token_for_user() {
+		$user_id = 1;
+
+		foreach ( $this->mock_payment_methods as $payment_method_id => $payment_method ) {
+			if ( 'card' === $payment_method_id ) {
+				$card_payment_method_mock = $this->array_to_object( self::MOCK_CARD_PAYMENT_METHOD_TEMPLATE );
+				$token = $payment_method->create_payment_token_for_user( $user_id, $card_payment_method_mock );
+				$this->assertTrue( 'WC_Payment_Token_CC' === get_class( $token ) );
+			} else {
+				$sepa_payment_method_mock = $this->array_to_object( self::MOCK_SEPA_PAYMENT_METHOD_TEMPLATE );
```

Comment:
```suggestion
			} else {
				$sepa_payment_method_mock = $this->array_to_object( self::MOCK_SEPA_PAYMENT_METHOD_TEMPLATE );
				if ( ! $payment_method->is_reusable() ) {
				    continue;
				}
```
This doesn't really break anything, but just worth noting that these tests only really apply to reusable payment methods. The old `add_token_from_payment_method` function returned `null` if the payment method was not reusable; I don't think it's crucial that this new function works the same, but just a thought that we don't really need to test how this function works with irrelevant payment methods.

Reply:
Oh, ok. I think we'd need to check the `$payment_method->is_reusable()`, though, right? `$sepa_payment_method_mock` doesn't have that property.

Reply:
Whoops, yeah, you're right. That should be `$payment_method->is_reusable()`. 🤦‍♂️